### ISSUE TITLE
New Feature: Count the # of Nodes Selected in Site Tree

### ISSUE THREAD

**Issue by davewichers** [zaproxy/zaproxy#4547]
One of the challenges of getting a good ZAP attack scan is first identifying all the endpoints in a web app. To help the analyst determine how much of the target site has been discovered by ZAP, I think it would be helpful to have a feature that allows a user to select a folder, or set of folders/nodes in the site tree, and then have ZAP count all the nodes in those part(s) of the tree on down.

Then later, when the user runs either Spider, or surfs the site by hand through ZAP, they could go back and recount to determine if they have discovered any additional endpoints or not. Note: I do like the idea of the new 'ATTACK" mode which causes ZAP to auto scan new nodes that have been discovered. I think that's very helpful, but it's not on by default.

**Comment by thc202** [comment]
Subset of #3738.

**Comment by davewichers** [comment]
Yes. You are right. This is a piece of what I requested before. I think it
is the easiest and most important piece. Any chance we can get just this
one part implemented? I can totally understand why #3738 has not been
addressed as its complicated.

**Comment by kingthorin** [comment]
> Any chance we can get just this one part implemented?

If someone decides to work on it.
